### DISCUSSION TITLE
How to download sympy on MSYS2？

### DISCUSSION THREAD

**Discussion by BaoSicheng** [sympy/sympy#27024] [General]
How to download sympy on MSYS2？I tried many ways but cannot get it done.

**Comment by lulunac27a** [comment]
First install MSYS2, then open the MSYS2 terminal and type `pacman -Syu` to update the package database, `pacman -S python` to install Python in MSYS2, and then `pip install sympy` to install SymPy on MSYS2.

**Comment by TomMonkeyMan** [comment]
Step-by-step guide:

Install MSYS2: If you haven’t already installed MSYS2, download it from the official [MSYS2 website](https://www.msys2.org/) and follow the installation instructions.
Update the package database: Open the MSYS2 terminal and update your system by running:
```
pacman -Syu
```
You may need to close and reopen the terminal after the first update.

Install Python in MSYS2: Ensure that you have Python installed within MSYS2. If not, install it using:
```
pacman -S mingw-w64-x86_64-python
```

Install pip for Python in MSYS2: If pip is not already installed, install it via:
```
pacman -S mingw-w64-x86_64-python-pip
```

Install SymPy via pip: Once Python and pip are installed, you can install SymPy using pip:
```
pip install sympy
```

Verify Installation: To confirm that SymPy is installed correctly, open the Python shell in MSYS2:
```
python
```

Then, try importing SymPy:
```
import sympy
```
If no error appears, SymPy is successfully installed.
